index.html

```diff
@@ -0,0 +1,40 @@
+<html>
+    <head>
+        <title>My Simple Base Page</title>
+        <link rel="stylesheet" type="text/css" href="css/main.css"/>
+        <link href="https://fonts.googleapis.com/css?family=Allan|Asap+Condensed&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <header class="navigation" >
+            <ul>
+               <li> <a href >Home</a></li>
+               <li> <a href>About me</a></li>
+               <li  class="base"> <a href="#index.html">Base Page</a></li>
```

Comment:
I would leave this href attr blank for now. 
`#index.html` because it starts with hash leads to something confusing like anchor tag with `id="index.html"` or something weird...